### DISCUSSION TITLE
Centroid Shape

### DISCUSSION THREAD

**Discussion by jakariaemon** [pyannote/pyannote-audio#1485] [Q&A] [answered]
In the earlier version, the centroid shape was 192. However, in Pyannote 3, it's 256.  
The pipeline still utilizes the same SpeechBrain embedding model with an embedding shape of 192. Am i missing something here? 
![image](https://github.com/pyannote/pyannote-audio/assets/43466665/4733bfa9-38ff-4135-9c3e-a698162d67b3)

**Comment by hbredin** [comment, marked as answer]
The `pyannote/speaker-diarization-3.0` pipeline does [**not**](https://huggingface.co/pyannote/speaker-diarization-3.0/blob/main/config.yaml#L7) use `speechbrain` model.

It relies on [hbredin/wespeaker-voxceleb-resnet34-LM](https://huggingface.co/hbredin/wespeaker-voxceleb-resnet34-LM).

You can still revert back to `speechbrain`'s if needed but you will have to tune the clustering threshold hyper-parameter.

**Reply by jakariaemon** [reply]
Thank you for the fast response. Solved.
